packages/admin-next/dashboard/src/v2-routes/reservations/reservation-detail/components/edit-reservation/edit-reservation-modal.tsx

```diff
@@ -0,0 +1,50 @@
+import { EditReservationForm } from "./components/edit-reservation-form"
+import { Heading } from "@medusajs/ui"
+import { RouteDrawer } from "../../../../../components/route-modal"
+import { useInventoryItem } from "../../../../../hooks/api/inventory"
+import { useParams } from "react-router-dom"
+import { useReservationItem } from "../../../../../hooks/api/reservations"
+import { useStockLocations } from "../../../../../hooks/api/stock-locations"
+import { useTranslation } from "react-i18next"
+
+export const ReservationEdit = () => {
+  const { id } = useParams()
+  const { t } = useTranslation()
+
+  const { reservation, isLoading, isError, error } = useReservationItem(id!)
```

Comment:
suggestion: use `isPending` here also, `isLoading` should be enough but `isPending` guarantees data